guides/common/modules/proc_listing-using-satellite-ansible-modules.adoc

```diff
@@ -0,0 +1,49 @@
+[id="listing-using-satellite-ansible-modules_{context}"]
+== Listing the {Project} Ansible Modules
+
+You can view the installed {Project} Ansible modules by listing the content of the following directory:
+
+ifeval::["{build}" == "satellite"]
+----
+# ls /usr/share/ansible/collections/ansible_collections/redhat/satellite/plugins/modules/
+----
+
+Alternatively, you can also see the complete list of {Project} Ansible modules and other related information at https://cloud.redhat.com/ansible/automation-hub/redhat/satellite/docs in the Automation Hub.
+
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+----
+# ls /usr/share/ansible/collections/ansible_collections/theforeman/foreman/plugins/modules/
+----
+
+Alternatively, you can also see the complete list of {Project} Ansible modules and other related information at https://galaxy.ansible.com/theforeman/foreman in the Ansible Galaxy.
+
+endif::[]
+
+
+[NOTE]
+====
+At the time of writing, the `ansible-doc -l` command does not list collections yet.
+====
+
+ifeval::["{build}" == "satellite"]
+
+All modules are in the `redhat.satellite` namespace and can be referred to as follows:
```

Comment:
```suggestion
All modules are in the `redhat.satellite` namespace and can be referred to in the format `redhat.satellite._module_name_`. For example, to display information on the `activation_key` module, enter the following command:
```
Add a little more context and implement the example

Reply:
Done